docs/content/guides/upgrade-and-migration/migrating-from-15.3-to-16.0/migrating-from-15.3-to-16.0.md

```diff
@@ -0,0 +1,416 @@
+---
+id: migrating-15.3-to-16.0
+title: Migrating from 15.3 to 16.0
+metaTitle: Migrating from 15.3 to 16.0 - JavaScript Data Grid | Handsontable
+description: Migrate from Handsontable 15.3 to Handsontable 16.0, released on [].
+permalink: /migration-from-15.3-to-16.0
+canonicalUrl: /migration-from-15.3-to-16.0
+pageClass: migration-guide
+react:
+  id: migrating-15.3-to-16.0-react
+  metaTitle: Migrate from 15.3 to 16.0 - React Data Grid | Handsontable
+angular:
+  id: migrating-15.3-to-16.0-angular
+  metaTitle: Migrate from 15.3 to 16.0 - Angular Data Grid | Handsontable
+searchCategory: Guides
+category: Upgrade and migration
+---
+
+# Migrate from 15.3 to 16.0
+
+Migrate from Handsontable 15.3 to Handsontable 16.0, released on [TODO].
+
+[[toc]]
+
+## 1. Introducing the new DOM structure
+
+In Handsontable 16.0, we changed how the table is mounted in the DOM. Previously, the container `<div>` you provided became the root element of the table. Now, that container acts as a mounting point, and Handsontable creates and injects its own root element inside it.
+
+Here's a side-by-side comparison of the old and new DOM structures:
+
+Old DOM structure:
+
+```
+body
+├── #example.ht-wrapper.handsontable
+│   ├── Focus catcher (top)
```

Comment:
Maybe: for a better clarity I would stick with either CSS classes or descriptive names. Mixing them might be confusing. Instead it could look like this:

body
├── #example.ht-wrapper.handsontable       // root container
│   ├── .htFocusCatcher                                   // Catching focus top-down

......